KuduSync.NET/Program.cs

```diff
@@ -25,6 +27,7 @@ class Program
             "</body>" +
             "</html>";
         private const string AppOfflineSetting = "SCM_CREATE_APP_OFFLINE";
+        private static readonly string AppOfflineSentinelDirectory = Path.Combine(Environment.GetEnvironmentVariable("HOME"), "site", "deployments");
```

Comment:
@suwatch, is this a reliable place you think to put the "commit" file?

Reply:
I would put in "data" folder